readthedocs/doc_builder/base.py

```diff
@@ -134,8 +134,9 @@ def create_index(self, extension='md', **__):
 familiar with Read the Docs.
                 """
 
-                index_file.write(index_text.format(dir=docs_dir, ext=extension))
-                index_file.close()
+                with open(index_filename, 'w+') as index_file:
```

Comment:
you should remove the open call at line ~120.

Reply:
Right. I missed that.